src/OpenLoco/src/Objects/VehicleObject.h

```diff
@@ -188,15 +188,21 @@ namespace OpenLoco
     enum class VehicleObjectFlags : uint16_t
     {
         none = 0U,
-        flag_02 = 1U << 2, // rollable? APT Passenger carriage
-        flag_03 = 1U << 3, // rollable? APT Driving carriage
+        flag_00 = 1U << 0, // All diesel and electric standalone locomotives except Deh 4/6, APT driving carriage, Eurostar power car
```

Comment:
There are one or two i named in this pr https://github.com/OpenLoco/OpenLoco/pull/2620/files could you review my names and integrate into this